x-pack/plugins/canvas/.storybook/webpack.config.js

```diff
@@ -5,19 +5,14 @@
  */
 
 const path = require('path');
-const TSDocgenPlugin = require('react-docgen-typescript-webpack-plugin');
+const webpack = require('webpack');
+const CopyWebpackPlugin = require('copy-webpack-plugin');
+const { DLL_OUTPUT, KIBANA_ROOT } = require('./constants');
 
-// Extend the Storybook Webpack config with some customizations;
-module.exports = async ({ config, _mode }) => {
-  // Include the React preset for Storybook JS files.
-  config.module.rules.push({
-    test: /\.js$/,
-    exclude: /node_modules/,
-    loaders: 'babel-loader',
-    options: {
-      presets: [require.resolve('@kbn/babel-preset/webpack_preset')],
-    },
-  });
+// Extend the Storybook Webpack config with some customizations
+module.exports = async ({ config }) => {
+  // Include timing and other data in the Webpack build stats file.
+  config.profile = true;
```

Comment:
Do you want to commit this? I am not sure what the overhead to profiling is.

Reply:
I can take it, sure.